<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLocalRefresh">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLocalBind">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Bind Device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLocalUnbind">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Unbind Device</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leIp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>10.10.1.177</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoteRefresh">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoteAttach">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote USB Devices for IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Local USB Devices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAttachedRefresh">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAttatchedDetatch">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Detach Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Attatched Devices</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tvLocal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>801</width>
      <height>192</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Bus_ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Manufacturer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tvRemote">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>192</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Bus_ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Manufacturer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tvAttached">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>801</width>
      <height>192</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Host</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Bus_ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Manufactoror</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionServer_Manager"/>
  </widget>
  <action name="actionServer_Manager">
   <property name="icon">
    <iconset>
     <normaloff>img/server-network-outline.svg</normaloff>img/server-network-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Server Manager</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnRemoteRefresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnRemoteRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoteAttach</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnRemoteAttach()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLocalRefresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnLocalRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLocalBind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnLocalBind()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLocalUnbind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnLocalUnbind()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAttachedRefresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnAttachedRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAttatchedDetatch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnAttachedDetach()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnLocalRefresh()</slot>
  <slot>btnLocalBind()</slot>
  <slot>btnRemoteRefresh()</slot>
  <slot>btnRemoteAttach()</slot>
  <slot>btnLocalUnbind()</slot>
  <slot>btnAttachedRefresh()</slot>
  <slot>btnAttachedDetach()</slot>
 </slots>
</ui>
